### ISSUE TITLE
roundBars not working in radial mode

### ISSUE THREAD

**Issue by Staijn1** [hvianna/audioMotion-analyzer#67]
In the fluid demo I noticed `roundBars` does not work when in `radial` mode.

I selected half octave / 20 bands mode, enabled round bars (working) and then enabled radial. Then the bars are no longer rounded.

Is this by design?

**Comment by hvianna** [comment]
Yeah, `roundBars` is barely noticeable on radial, but it should be working.

| roundBars ON | roundBars OFF |
|----|----|
| ![image](https://github.com/hvianna/audioMotion-analyzer/assets/1033735/cad53db3-4466-4ea1-8c99-92db92074fcf) | ![image](https://github.com/hvianna/audioMotion-analyzer/assets/1033735/b8619639-f909-4e0c-b15f-8e7b7b15cfd0) |
